src/wifi.rs

```diff
@@ -439,6 +468,11 @@ impl<'d> WifiDriver<'d> {
         Ok(caps)
     }
 
+    #[cfg(feature = "async")]
```

Comment:
Remove the `async` feature protection and just use plain old `pub async fn get_capabilities`. Note also that I suggested these functions to be on the `AsyncWifiDriver`, not on the `WifiDriver`. So you don't need the `_async` suffix as you won't have name clashes once you move them.